ledger-api/grpc-definitions/com/daml/ledger/api/v1/transaction_service.proto

```diff
@@ -39,6 +40,12 @@ service TransactionService {
   // Lookup a transaction by its ID.
   rpc GetFlatTransactionById (GetTransactionByIdRequest) returns (GetFlatTransactionResponse);
 
+  // Lookup events involving the provided contract ID
+  rpc GetContractById (GetContractByIdRequest) returns (GetContractByIdResponse);
```

Comment:
```suggestion
  rpc GetEventsByContractId (GetEventsByContractIdRequest) returns (GetEventsByContractIdResponse);
```
We are asking for events actually, not for contracts.
We should not fall it FlatTransactions nor TransactionTrees because what we are getting back is different from both flats and trees